lib/shopify_transporter/exporters/magento/customer_exporter.rb

```diff
@@ -12,14 +12,21 @@ def initialize(store_id: nil, soap_client: nil, database_adapter: nil)
 
         def export
```

Comment:
All three `SomethingExporter#export` functions are nearly the same, so there's potential for some OOP stuff. I'm leaving it simple for now and we can do some refactoring and stuff when the dust settles.